Add FAQ on omitted properties (#7190)
segmentio/segment-docs: src/protocols/faq.md

Blocked events are blocked from sending to all Segment Destinations, including warehouses and streaming Destinations. When an Event is blocked using a Tracking Plan, it does not count towards your MTU limit. They will, however, count toward your MTU limit if you enable [blocked event forwarding](/docs/protocols/enforce/forward-blocked-events/) in your Source settings.

### If I omit unplanned properties or properties that generate JSON schema violations, what happens to them?

> Properties that are omitted due to JSON Schema Violations or being unplanned are not stored in Segment logs. Segment only stores fully blocked events for 30 days. Omitted properties are dropped from the events and can only be found in the `context.violations` object of an event payload. If Violations are forwarded to a new source, then the omitted properties can also be seen in the Violation Generated event under `violationField` in the `properties` object of the event. 

Reviewer: ```suggestion
Segment doesn't store unplanned properties and properties omitted due to JSON Schema Violations in Segment logs. Omitted properties are dropped from the events and can only be found in the `context.violations` object of an event payload. If Violations are forwarded to a new source, then the omitted properties can also be seen in the Violation Generated event under `violationField` in the `properties` object of the event. 

Segment only stores fully blocked events for 30 days. 
```
